### PR TITLE
Updates to README

### PR DESCRIPTION
**Pull request by pmorie**
To more accurately reflect the current state of the repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,11 +9,11 @@
 The service-catalog project is in incubation to bring integration with service
 brokers to the Kubernetes ecosystem via the [Open Service Broker
 API](https://github.com/openservicebrokerapi/servicebroker). A service broker
-is an endpoint that manages a set of services.  The end-goal of the service-
-catalog project is to provide a way for Kubernetes users to consume services
-from brokers and easily configure their applications to use those services,
-without needing detailed knowledge about how those services are created /
-managed.
+is an endpoint that manages a set of capabilities (called services).  The end-
```
Comment (kibbles-n-bytes) [inline README.md]: "capabilities called services" seems very strange to me. Services aren't "capabilities"; capability is the _power/ability_ to do something, while services are external resources that provide clients their capabilities.

I would just substitute "capabilities" with "resources".

File: README.md [hunk 2]
```diff
@@ -9,11 +9,11 @@
...
+goal of the service- catalog project is to provide a way for Kubernetes users
```
Comment (jpeeler) [inline README.md]: extra space in service-catalog?

Comment (pmorie) [reply README.md]: Thanks

File: README.md (hunk 1 above)
Comment (pmorie) [reply README.md]: How about 'managed software offerings called services', which is how the spec defines the term.

Comment (kibbles-n-bytes) [reply README.md]: That sounds better to me. We already say it "manages" them, though, so I think just "software offerings called services" is good.

**Pull request merged**